commerce-product-api/src/main/java/com/liferay/commerce/product/service/CPAttachmentFileEntryLocalService.java

```diff
@@ -411,6 +411,34 @@ public CPAttachmentFileEntry updateStatus(
 			Map<String, Serializable> workflowContext)
 		throws PortalException;
 
+	/**
+	 * @param classNameId
+	 * @param classPK
+	 * @param fileEntryId
+	 * @param displayDateMonth
+	 * @param displayDateDay
+	 * @param displayDateYear
+	 * @param displayDateHour
+	 * @param displayDateMinute
+	 * @param expirationDateMonth
+	 * @param expirationDateDay
+	 * @param expirationDateYear
+	 * @param expirationDateHour
+	 * @param expirationDateMinute
+	 * @param neverExpire
+	 * @param titleMap
+	 * @param json
+	 * @param priority
+	 * @param type
+	 * @param externalReferenceCode
+	 * @param serviceContext
+	 * @throws PortalException
+	 * @deprecated As of Athanasius (7.3.x), use {@link
+	 #upsertCPAttachmentFileEntry(long, long, long, long, int,
+	 int, int, int, int, int, int, int, int, int, boolean, Map,
+	 String, double, int, String, ServiceContext)}
```

Comment:
@igorbeslic thank you for this -- deprecation that includes an indication of what to do instead is ❤️